assets/src/blocks/Covers/CoversCarouselButton.js

```diff
@@ -0,0 +1,8 @@
+const { __ } = wp.i18n;
+
+export const CoversCarouselButton = ({ handler, direction, disabled = false }) => (
+  <button className={`carousel-control-${direction}`} disabled={disabled} role="button" onClick={handler}>
```

Comment:
You don't need to set `role="button"` to any `button` since it has already set this value by default.

Take a look the column from here https://www.w3.org/TR/2014/REC-html5-20141028/dom.html#aria-role-attribute
**Language feature**: `button` element 
**Default implicit ARIA semantic**: `button` role

Reply:
Removed 👍